tests/test_adjointA.py

```diff
@@ -17,13 +17,13 @@
 @skipif_yask
 @pytest.mark.parametrize('mkey, shape, kernel, space_order, nbpml', [
     # 2D tests with varying time and space orders
-    ('layers', (60, 70), 'OT2', 4, 10), ('layers', (60, 70), 'OT2', 8, 10),
-    ('layers', (60, 70), 'OT2', 12, 10), ('layers', (60, 70), 'OT4', 4, 10),
-    ('layers', (60, 70), 'OT4', 8, 10), ('layers', (60, 70), 'OT4', 12, 10),
+    ('layers', (60, 70), 'OT2', 4, 20), ('layers', (60, 70), 'OT2', 8, 20),
```

Comment:
why?

Reply:
I don't like pml not bigger than stencil radius. produces odd results.